test/vm.js

```diff
@@ -406,6 +406,30 @@ describe('VM', () => {
       vm.getSerialPortOutput(port, assert.ifError);
     });
 
+    it('should override the start', done => {
+      const start = 10;
+
+      FakeServiceObject.prototype.request = function (reqOpts) {
+        assert.strictEqual(reqOpts.qs.start, start);
+        done();
+      };
+
+      vm.getSerialPortOutput({start}, assert.ifError);
+    });
+
+    it('should override the port and start', done => {
```

Comment:
I think we can remove this test, since we already tested these options individually before.

Reply:
Done!